### DISCUSSION TITLE
`self-baseline` utility?

### DISCUSSION THREAD

**Discussion by nwhittaker** [tailwindlabs/tailwindcss#4531] [Ideas]
Any appetite for adding an entry to [`plugins/alignSelf.js`](https://github.com/tailwindlabs/tailwindcss/blob/0a558598670fb4c9bd91282add1100a86accc206/src/plugins/alignSelf.js) for the `align-self: baseline` style?

```js
'.self-baseline': {
  'align-self': 'baseline',
},
```

I'm using a web component library where I can style the flex children of an element, but not the parent, and had a need to align some flex items by their baselines.

**Comment by hacknug** [comment]
@adamwathan any chance we can sneak this one into a minor update? I can open a PR with the changes and tests if you want me to :+1:

**Reply by adamwathan** [reply]
Yep I'm good with it!

**Reply by hacknug** [reply]
@nwhittaker #5000 has been merged. I'm guessing the next release will include your suggestion 💅
